.github/workflows/haskell.yaml

```diff
@@ -0,0 +1,68 @@
+name: Haskell
+on:
+  workflow_dispatch:
+  push:
+    branches:
+      - master
+    tags:
+      - "*"
+  pull_request:
+    paths:
+      - "haskell/**"
+      - .github/workflows/haskell.yaml
+jobs:
+  build:
+    name: Build
+    runs-on: ubuntu-latest
```

Comment:
We should pin this to ubuntu-1804 since the haskell binary is not fully static, otherwise we'll produce an artifact that'll only run on ubuntu 20.04 once `ubuntu-latest` switches:

```
❯ ldd sbp2prettyjson                                                                                                                                        (base)
        linux-vdso.so.1 (0x00007ffd42b6f000)
        libm.so.6 => /lib/x86_64-linux-gnu/libm.so.6 (0x00007f535776d000)
        libz.so.1 => /lib/x86_64-linux-gnu/libz.so.1 (0x00007f5357751000)
        librt.so.1 => /lib/x86_64-linux-gnu/librt.so.1 (0x00007f5357746000)
        libutil.so.1 => /lib/x86_64-linux-gnu/libutil.so.1 (0x00007f5357741000)
        libdl.so.2 => /lib/x86_64-linux-gnu/libdl.so.2 (0x00007f535773b000)
        libgmp.so.10 => /lib/x86_64-linux-gnu/libgmp.so.10 (0x00007f53576b7000)
        libpthread.so.0 => /lib/x86_64-linux-gnu/libpthread.so.0 (0x00007f5357692000)
        libgcc_s.so.1 => /lib/x86_64-linux-gnu/libgcc_s.so.1 (0x00007f5357677000)
        libc.so.6 => /lib/x86_64-linux-gnu/libc.so.6 (0x00007f5357485000)
        /lib64/ld-linux-x86-64.so.2 (0x00007f53578c9000)
 ```

Reply:
Actually looks like this already happened: https://github.blog/changelog/2020-10-29-github-actions-ubuntu-latest-workflows-will-use-ubuntu-20-04/